Meetups.elm

```diff
@@ -34,19 +34,19 @@ meetup1 =
     { date = "2016-03-24"
     , location = "Meetic"
     , lineUp =
-        [ { speaker = tbrisbout
+        [ { speakers = [ tbrisbout ]
```

Comment:
Single speaker talks would need to be changed as in:
```elm
        [ { speakers = (tbrisbout, [])
```
Whereas multiple-speakers talks would go like this:
```elm
        [ { speakers = (sebcreme, [ erwan ])
```